lib/analyse.ml

```diff
@@ -226,6 +228,22 @@ module Analysis = struct
     let selection = List.hd selections in
     (selection.Selection.commit, selection)
 
+  let get_lower_bound_platforms platforms =
+    List.filter
+      (fun (v, _) -> String.equal (Variant.distro v) "debian-11")
```

Comment:
This debian-11 should use the distro information from `ocaml-dockerfile`
```
 `Debian `Latest
```
That way it will roll forward when new versions are released.